package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "react-targem",
+  "version": "0.0.0",
+  "description": "A React library for efficiently implementing Gettext localization",
+  "main": "./dist/bundles/index.umd.js",
+  "module": "./dist/esm5/index.js",
+  "es2015": "./dist/esm2015/index.js",
+  "typings": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trucknet-io/react-targem.git"
+  },
+  "keywords": [
+    "react",
+    "targem",
+    "i18n",
+    "typescript",
+    "localization"
+  ],
+  "author": "goooseman, alexeychikk",
+  "license": "MIT",
+  "scripts": {
+    "pretest": "npm run import-strings",
+    "prestart": "npm run import-strings",
+    "prebuild": "npm run import-strings",
+    "prelint": "npm run import-strings",
+    "precommit": "npm run export-strings && git add dev/src/i18n/*.po* && lint-staged && npm run lint",
+    "export-strings": "gettext-utils export-strings 'dev/src/**/{*.ts,*.tsx}' dev/src/i18n/template.pot --default-locale=en-GB",
+    "import-strings": "gettext-utils import-strings dev/src/i18n dev/src/i18n/translations.json",
+    "commit": "commit-prompt --no-verify",
+    "lint": "npm run lint:tsc && npm run lint:tslint",
+    "lint:tsc": "tsc -p dev/tsconfig.json --noEmit",
+    "lint:prettier": "prettier --write \"{src,dev/src,test}/**/*.ts*\"",
+    "lint:tslint": "tslint -c tslint.json -p ./dev/tsconfig.json --fix",
+    "start": "env-cmd development webpack-dev-server --config dev/webpack.config.js --hot",
+    "build": "rimraf 'dist' && npm run build:all",
+    "build:all": "tsc && tsc --target es2018 --outDir dist/esm2015 && rollup -c config/rollup.config.js && rollup -c config/rollup.config.js --environment NODE_ENV:production",
```

Comment:
we have a `memoize-one` dependency

And we are building a bundle.

So what happens, when I import this lib to another project, which uses webpack to build the bundle? Will be memoize-one inside the bundle already or it will be taken from node_modules of the root project?

Reply:
@goooseman mjs/umd bundles are only used when you load them as `<script type="module" />` or in the Node.js with `--experimental-modules` flag.
When you do `import { TargemProvider } from "react-targem"` in webpack project, it loads all `react-targem` dependencies (lodash.get, memoize-one) from your node_modules, because they were already installed when you did `npm i react-targem`.